DS and Algorithm/bubblesort.py

```diff
@@ -1,3 +1,15 @@
+# Write a bubble sort for python code 
+def bubble_sort(arr):
+    n = len(arr)
+    # Traverse through all elements in the array
+    for i in range(n):
+        # Last i elements are already in place
+        for j in range(0, n-i-1):
+            # Traverse the array from 0 to n-i-1
+            # Swap if the element found is greater than the next element
+            if arr[j] > arr[j+1]:
+                arr[j], arr[j+1] = arr[j+1], arr[j]
```

Comment:
**Scalability Issue**: Bubble sort is inherently not scalable due to its O(n^2) time complexity, making it inefficient for large arrays. <br> **Fix**: Replace bubble sort with a more efficient algorithm like heapsort or quicksort for scalability. <br> **Code Suggestion**: 
 ```
 +def quick_sort(arr):
 +    if len(arr) <= 1:
 +        return arr
 +    else:
 +        pivot = arr[len(arr) // 2]
 +        left = [x for x in arr if x < pivot]
 +        middle = [x for x in arr if x == pivot]
 +        right = [x for x in arr if x > pivot]
 +        return quick_sort(left) + middle + quick_sort(right)
 ```